zombie-bite/README.md

```diff
@@ -0,0 +1,45 @@
+# Zombie-bite
+
+## Intro
+
+`zombie-bite` is an cli tool that allow you to _fork and spawn_ live networks (e.g polkadot/kusama) keeping the _live state_ with the needed customizations in order to make the new chain/s keep progressing.
+
+### Instruction to spawn Polkadot(with sudo)/AH
+
+ - Install `zombie-bite`
+
+   ```
+   cargo install --git https://github.com/pepoviola/zombie-bite --bin zombie-bite
+   ```
+
+- Patch and compile polkadot runtime to add sudo
+
+Apply the patch `polkadot_sudo.patch` on top of the polkadot code in `runtimes` directory and then compile the runtime.
+
+```
+cd ../runtimes && cargo build --release -p polkadot-runtime && cd -
```

Comment:
```suggestion
cd ../runtimes
git apply ../zombie-bite/polkadot_sudo.patch
cargo build --release -p polkadot-runtime
cd -
```
We can also add this to the justfile at the point we compile the runtime [here](https://github.com/paritytech/ahm-dryrun/blob/main/justfile#L44)